kratos/python_scripts/multistage_orchestrators/sequential_multistage_orchestrator.py

```diff
@@ -0,0 +1,175 @@
+
+class SequentialMultistageOrchestrator(MultistageOrchestrator):
+
+    def __init__(self, settings : KratosMultiphysics.Parameters) -> None:
+        super().__init__(settings)
+
+    def Run(self):
+        '''Main function that runs a sequential multistage simulation.'''
+
+        # Set the stages to be saved first
+        # We do it first to warn the user about the non-present stages before running the first one
+        self.__GetStagesToCheckpointList()
+
+        # Check if loading from checkpoint is required and load
+        if self.GetProject().GetSettings()["orchestrator"]["settings"].Has("load_from_checkpoint"):
+            if not self.GetProject().GetSettings()["orchestrator"]["settings"]["load_from_checkpoint"].IsNull():
+                if not self.GetProject().GetSettings()["orchestrator"]["settings"]["load_from_checkpoint"].IsString():
+                    err_msg = "'load_from_checkpoint' is expected to be null or a string with the loading point."
+                    raise TypeError(err_msg)
+                else:
+                    loading_point = self.GetProject().GetSettings()["orchestrator"]["settings"]["load_from_checkpoint"].GetString()
+                    self.GetProject().Load(loading_point)
+
+        # Run the stages list
+        for stage_name in self.__GetExecutionList():
+            # Check current stage settings
+            self.CheckStageSettings(stage_name)
+
+            # Set up and check current stage instance
+            current_stage = self.CreateStage(stage_name)
+
+            # Execute current stage preprocess
+            self.RunCurrentStagePreprocess(stage_name)
+
+            # Run (solve) current stage
+            current_stage.Run()
+
+            # Get the final data dictionary
+            self.GetProject().GetOutputData()[stage_name] = current_stage.GetFinalData()
+
+            # Execute current stage postprocess
+            self.RunCurrentStagePostprocess(stage_name)
+
+            # Delete current stage instance
+            del current_stage
+
+            # Output validated simulation settings (with defaults) as simulation report
+            output_settings_name = self.__PrepareOutputSettings(stage_name)
+
+            # Check the current stage is to be checkpointed and save it if so
+            if stage_name in self.__GetStagesToCheckpointList():
+                save_folder_name = self.__GetStagesCheckpointFolder()
+                self.GetProject().Save(save_folder_name, stage_name, output_settings_name)
+
+
+    def GetNumberOfStages(self):
+        '''Returns the number of stages.'''
+
+        return len(self.__GetExecutionList())
+
+    def __GetExecutionList(self):
+        '''Creates and returns the execution list.
+        This method creates the execution list, either from a user-defined execution list or,
+        if this is not provided, from the stages declaration order.
+        '''
```

Comment:
If it creates I would not call it Get....

Reply:
So I would correct the documentation

Reply:
Done.